### PR TITLE
Change RPMs for 1.1.8

### PR DESCRIPTION
**Pull request by brian-avery**

### CODE REVIEW

File: istio/istio.spec [hunk 1]
```diff
@@ -14,7 +14,7 @@
 %global debug_package   %{nil}
 %endif
 
-%global git_commit c5db59a9076a5b7bf968ae8f69cc786b3e015d78
+%global git_commit fd5254d87df6733d09972d53d532564d6c499c2b
```
Comment (jwendell) [inline istio/istio.spec]: This is wrong. It should be `7d3618ce62aed95e4d1d064af6d237d3311aff30`.

**Review by jwendell — CHANGES_REQUESTED** [review]

File: istio/istio.spec (hunk 1 above)
Comment (brian-avery) [inline istio/istio.spec]:
```suggestion
%global git_commit 7d3618ce62aed95e4d1d064af6d237d3311aff30
```

Comment (brian-avery) [reply istio/istio.spec]: Good catch. Thanks. I just pushed that to product. let me fix it.

Comment (brian-avery) [reply istio/istio.spec]: Interesting. I had the right one there, but wrong here.

File: istio/sources [hunk 2]
```diff
@@ -1 +1 @@
-e3dd022371d844764704ae17e74f1ff4  istio-c5db59a9076a5b7bf968ae8f69cc786b3e015d78.tar.gz
+47f6e9d7850ec3892bc6120592704c4e  istio-fd5254d87df6733d09972d53d532564d6c499c2b.tar.gz
```
Comment (jwendell) [inline istio/sources]: This also needs to be changed

**Review by jwendell — CHANGES_REQUESTED** [review]

**Review by jwendell — APPROVED** [review]

**Review by rcernich — APPROVED** [review]

**Pull request merged**